src/components/shared/form_fields/useCityAutocompleteEvents.ts

```diff
@@ -1,30 +0,0 @@
-import { nextTick, Ref, ref } from 'vue';
-
-export function useCityAutocompleteEvents( city: Ref<string>, emit: ( event: string ) => void ) {
-	const autocompleteIsActive = ref<Boolean>( false );
-
-	const onFocus = ( event: Event ) => {
-		autocompleteIsActive.value = true;
```

Comment:
what was the reason for deleting the Composables and integrating the behaviour back into the classes?

Reply:
The 2 autocomplete components were originally using a shared composable, which ended up being split into separate ones for each. This meant each composable was only being used by one component and it was mentioned in the comments of an earlier PR (which I can't find now) that they were needless abstractions.

So while I was working on those fields anyway I integrated them back in.